### ISSUE TITLE
Alert-Modul; custom.css is overridden.

### ISSUE THREAD

**Issue by desq42** [MichMich/MagicMirror#284]
The custom.css-file is overridden by "ns-default.css", so it´s not possible to style Alert-Notifications…
![overridden](https://cloud.githubusercontent.com/assets/18470786/14986705/529e4736-114d-11e6-996d-2e4e9073f2a6.png)

**Comment by MichMich** [comment]
Fixed! Could you give it a try?

**Comment by desq42** [comment]
Tested & Approved. Thank you.
![bildschirmfoto 2016-05-04 um 10 15 16](https://cloud.githubusercontent.com/assets/18470786/15008550/4f821d86-11e1-11e6-9a9c-7e6048d6f96c.png)

**Comment by Varguit** [comment]
Hi, I'm trying to increase the font-size of the alert, I want it to fit almost the entire screen.
 
I've copied the css to custom and modified the font-size that comes at 70% by default, I've tried with 300% and even with 300px but no effect on the size of the text.
Can someone tell me how to? I don't want to modify de original alert.js where the font-size is 30px for the title and 22px for the message.

That's my custom.css

`.ns-box {

  background: #fff;
  padding: 17px;
  line-height: 1.4;
  margin-bottom: 10px;
  z-index: 1;
  color: black;
  **font-size: 300%;**
  position: relative;
  display: table;
  word-wrap: break-word;
  max-width: 100%;
  border-radius: 5px;
}

.ns-alert {
  border-style: solid;
  border-color: #fff;
  padding: 17px;
  line-height: 1.4;
  margin-bottom: 10px;
  z-index: 3;
  color: white;
  **font-size: 300%;**
  position: fixed;
  text-align: center;
  right: 0;
  left: 0;
  margin-right: auto;
  margin-left: auto;
  top: 40%;
  width: 40%;
  height: auto;
  word-wrap: break-word;
  border-radius: 20px;
}`
